suite.go

```diff
@@ -27,16 +27,26 @@ type SuiteOpts struct {
 	Skip bool
 	// whether to show daemon logs
 	Verbose bool
+	// Whether to spawn child containers in parallel. This doesn't spawn
+	// all containers in parallel, only the ones that are on the same hierarchy level, e.g.:
+	//  // c1 and c2 are started in parallel after the network
+	//  network.After(c1)
+	//  network.After(c2)
+	//  // c3 and c4 are started in parallel after c1
+	//  c1.After(c3)
+	//  c1.After(c4)
+	SpawnParallel bool
```

Comment:
I have no strong opinion on this functionality. It decreased testing suite time of cmtd though by around 5 seconds.

Reply:
Im saying that parallel spawn should be default behavior because in 95% cases you would like to make it as fast as possible. Question from my side is. How this behavior should be controlled, using `SuiteOpts` or global variable through flags.

Reply:
Maybe, shouldn't we make verbosity global too then? It's unlikely people want to micro-manage it, I guess.